basics/patching.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: contribute
+title: Patch acceptance process
+---
+
+# Patch Acceptance Process
+
+1. Read the [Bazel governance plan](/governance.html).
+1. Discuss your plan and design, and get agreement on a GitHub issue or our
+   [mailing list](https://groups.google.com/forum/#!forum/bazel-dev).
+1. PRs that change or add behavior are not accepted without being tied to an
+   issue.
+1. Significant changes need a [design document](/designs/index.html).
+1. Prepare a git commit that implements the feature. Don't forget to add tests
+   and update the documentation.
+1. Ensure you've signed a [Contributor License
+   Agreement](https://cla.developers.google.com).
+1. Send us a pull request on
+   [GitHub](https://github.com/bazelbuild/bazel/pulls). If you're new to GitHub,
+   read [about pull
+   requests](https://help.github.com/articles/about-pull-requests/). Note that
+   we restrict permissions to create branches on the main Bazel repository, so
+   you will need to push your commit to [your own fork of the
+   repository](https://help.github.com/articles/working-with-forks/).
+1. Wait for a Bazel team member to assign you a reviewer. It should be done in 2
```

Comment:
Are we using "Bazel team member" as a synonym for "core contributor"? Just trying to figure out the appropriate terminology.

Reply:
Later we use the phrase "Bazel maintainer at Google". Can we consolidate terms?

Reply:
Used maintainer throughout.